### DISCUSSION TITLE
Get size of kitty panel

### DISCUSSION THREAD

**Discussion by partisani** [kovidgoyal/kitty#8114] [Q&A]
I can't find a way to get the size of a panel i started.

I tried:

* the command: `tput cols` (returns the size of the parent terminal)
* the nushell built-in: `(term size).columns` (returns the size of the parent terminal)

**Comment by kovidgoyal** [comment]
what panel? kitty has no panels.

**Reply by partisani** [reply]
<sup>the `panel` kitten</sup>
<sup>but sadly, i don't use `kitty` anymore</sup>
